### ISSUE TITLE
Conflicting cross-version in dependencies

### ISSUE THREAD

**Issue by alonsodomin** [lucidsoftware/neo-sbt-scalafmt#3]
Getting the following error when using this SBT plugin:

```
[error] Modules were resolved with conflicting cross-version suffixes in {file:/home/travis/build/alonsodomin/cron4s/project/}cron4s-build:
[error]    com.lihaoyi:sourcecode _2.10, _2.11
```

The project itself has a dependency on `fastparse`, which will bring that `sourcecode` transitively. The full build log where this is thrown can be seen in Travis:
https://travis-ci.org/alonsodomin/cron4s/jobs/236165280

Source code is available in https://github.com/alonsodomin/cron4s, branch `scalafmt`.

**Comment by pauldraper** [comment]
Had some bad dependencies being added becuase sbt-scalafmt was itself using an old version sbt-scalafmt.

Fixed in sbt-scalafmt 1.0.

**Comment by oripwk** [comment]
@pauldraper 

Hi, it happens in 1.16 on clean installation
